package.json

```diff
@@ -41,7 +41,7 @@
     "prettier": "2.3.1",
     "serverless": "^2.47.0",
     "serverless-api-gateway-throttling": "^1.1.0",
-    "serverless-dynamodb-local": "^0.2.39",
+    "serverless-dynamodb-local": "^0.2.40",
```

Comment:
Hey @alpeshvas, just enabled the [workflow](https://github.com/connorads/lockbot/runs/4872400936?check_suite_focus=true) on this PR 🙃 
You'll also need to commit the corresponding changes to `yarn.lock`
![image](https://user-images.githubusercontent.com/10026538/150194391-f3ceb764-298d-4eaa-8247-25b82643cbdb.png)

Reply:
Done, I think your approval is required to re-trigger the workflow.

Reply:
> Hey @alpeshvas, just enabled the [workflow](https://github.com/connorads/lockbot/runs/4872400936?check_suite_focus=true) on this PR upside_down_face You'll also need to commit the corresponding changes to `yarn.lock` ![image](https://user-images.githubusercontent.com/10026538/150194391-f3ceb764-298d-4eaa-8247-25b82643cbdb.png)


Done. Also, we can use something like this for auto generation of swagger , wdyt @connorads ?https://www.serverless.com/plugins/serverless-openapi-documentation

Reply:
Not sure what's happened with `yarn.lock` here but the changes should be much smaller given you've only bumped the version of `serverless-dynamodb-local` in `package.json`
![image](https://user-images.githubusercontent.com/10026538/150840823-437f9b34-8965-4a3e-be18-6a98cfa9c1d4.png)

```
serverless-dynamodb-local@^0.2.40:
  version "0.2.40"
  resolved "https://registry.yarnpkg.com/serverless-dynamodb-local/-/serverless-dynamodb-local-0.2.40.tgz#1c97f678244bfb581e5850d6d43335d3131b59ee"
  integrity sha512-8wPtFxeJnRKTN3u8x1jG7Cs5VlkO0GkUPfb+DCKIDzjGM5n+qjHXoVsr4YZ/TPwET9kMUgH1Jv39oaEyeSJvYQ==
```

PS. Why did you need to bump the version?

Reply:
> @connorads: "PS. Why did you need to bump the version?"
I think I know the answer to this now, I had an error with deploying so I've also upgraded it and merged it into `master`

Meaning this PR now has conflicts.
![image](https://user-images.githubusercontent.com/10026538/151680310-b9b9e58f-ac49-4b96-9700-a02f079e5065.png)

In this PR we just need to take the changes from `master` - where I've also upgraded a few other packages.